zebra-state/src/response.rs

```diff
@@ -32,6 +32,9 @@ pub enum Response {
     /// Response to [`Request::Block`] with the specified block.
     Block(Option<Arc<Block>>),
 
-    /// The response to a `AwaitUtxo` request
+    /// The response to a `AwaitUtxo` request.
     Utxo(transparent::Output),
+
+    /// The response to a `GetHashes` request.
+    Hashes(Option<Vec<block::Hash>>),
```

Comment:
I don't think you need an `Option` here - what's the difference between an empty vector and `None`?

Modified from:
https://github.com/ZcashFoundation/zebra/blob/main/zebra-network/src/protocol/internal/response.rs#L21

```suggestion
    /// The response to a `FindBlockHashes` request.
    BlockHashes(Vec<block::Hash>),
```